### ISSUE TITLE
Uncaught TypeError: Cannot read property 'null' of undefined

### ISSUE THREAD

**Issue by TommyM** [lloiser/go-debug#24]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.8.0-beta2
**System**: Mac OS X 10.11.4
**Thrown From**: [go-debug](https://github.com/lloiser/go-debug) package, v0.1.11

### Stack Trace

Uncaught TypeError: Cannot read property 'null' of undefined

```
At /Users/tmaintz/.atom/packages/go-debug/node_modules/react-for-atom/node_modules/react/lib/ReactMount.js:649

TypeError: Cannot read property 'null' of undefined
    at Object.ReactMount.findReactContainerForID (/Users/tmaintz/.atom/packages/go-debug/node_modules/react-for-atom/node_modules/react/lib/ReactMount.js:649:50)
    at findParent (/Users/tmaintz/.atom/packages/go-debug/node_modules/react-for-atom/node_modules/react/lib/ReactEventListener.js:41:30)
    at handleTopLevelWithoutPath (/Users/tmaintz/.atom/packages/go-debug/node_modules/react-for-atom/node_modules/react/lib/ReactEventListener.js:87:16)
    at handleTopLevelImpl (/Users/tmaintz/.atom/packages/go-debug/node_modules/react-for-atom/node_modules/react/lib/ReactEventListener.js:73:3)
    at ReactDefaultBatchingStrategyTransaction.Mixin.perform (/Users/tmaintz/.atom/packages/go-debug/node_modules/react-for-atom/node_modules/react/lib/Transaction.js:136:20)
    at Object.ReactDefaultBatchingStrategy.batchedUpdates (/Users/tmaintz/.atom/packages/go-debug/node_modules/react-for-atom/node_modules/react/lib/ReactDefaultBatchingStrategy.js:62:19)
    at Object.batchedUpdates (/Users/tmaintz/.atom/packages/go-debug/node_modules/react-for-atom/node_modules/react/lib/ReactUpdates.js:94:20)
    at ReactEventListener.dispatchEvent (/Users/tmaintz/.atom/packages/go-debug/node_modules/react-for-atom/node_modules/react/lib/ReactEventListener.js:204:20)
```

### Commands

```
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "atom-wallaby"
    ],
    "packagesWithKeymapsDisabled": [
      "lcov-info"
    ],
    "themes": [
      "brogrammer-ui",
      "bronokai"
    ]
  },
  "go-debug": {
    "limitCommandsToGo": true,
    "panelInitialVisible": false
  }
}
```

### Installed Packages

```coffee
# User
atom-typescript, v8.8.1
autocomplete-go, v1.0.6
brogrammer-ui, v1.0.0
bronokai, v1.0.0
builder-go, v1.0.0
docblockr, v0.7.3
editorconfig, v1.4.0
environment, v1.2.0
file-icons, v1.7.6
go-config, v1.1.4
go-debug, v0.1.11
go-get, v1.0.3
go-plus, v4.1.0
go-quick-import, v0.2.0
go-signature-statusbar, v1.0.2
godoc, v0.2.0
gofmt, v1.1.6
gometalinter-linter, v1.1.0
gorename, v1.0.2
language-proto, v0.7.0
language-terraform, v0.7.4
linter, v1.11.4
linter-eslint, v7.2.1
navigator-godef, v1.0.3
pretty-json, v1.0.3
project-manager, v2.9.7
project-view, v0.5.0
ssh-config, v0.10.1
tester-go, v1.0.4

# Dev
No dev packages
```

**Comment by lloiser** [comment]
What did you do? How can I reproduce this?

**Comment by TommyM** [comment]
Sorry for not providing more info when I opened the issue. I was in a bit of a hurry.

Basically this started happening when I open non-go source files after I updated to the latest go-debug version this morning. The annoying thing is that it keeps throwing the error (and thus showing an error notification) every few seconds.

**Comment by lloiser** [comment]
seems to be fixed with the new react version! (a4e227bc692114afec8a1881f99535f3c541e94e)
